### PR TITLE
remove redundant accumulation normalization in manual optimization

### PR DESCRIPTION
**Pull request by awaelchli**
## What does this PR do?

Fixes #\<issue_number>

## Before submitting

- [ ] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [ ] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [ ] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [ ] Did you make sure to **update the documentation** with your changes? (if necessary)
- [ ] Did you write any **new necessary tests**? (not for typos and docs)
- [ ] Did you verify new and **existing tests pass** locally with your changes?
- [ ] Did you list all the **breaking changes** introduced by this pull request?
- [ ] Did you **update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)**? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [ ] Is this pull request ready for review? (if not, please submit in draft mode)
- [ ] Check that all items from **Before submitting** are resolved
- [ ] Make sure the title is self-explanatory and the description concisely explains the PR
- [ ] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

File: pytorch_lightning/loops/optimization/manual_loop.py [hunk 1]
```diff
@@ -55,7 +55,6 @@ def from_training_step_output(
             )
 
         if "loss" in extra:
-            # accumulate the loss. If `accumulate_grad_batches == 1`, no effect.
             # we detach manually as it's expected that it will have a `grad_fn`
             extra["loss"] = extra["loss"].detach().div(normalize)
```
Comment (carmocca) [inline pytorch_lightning/loops/optimization/manual_loop.py:59]: remove the `div` and input arg too?

Comment (awaelchli) [reply pytorch_lightning/loops/optimization/manual_loop.py:59]: I guess it's optional, I don't care. someone can use it if they want.

Comment (awaelchli) [reply pytorch_lightning/loops/optimization/manual_loop.py:59]: what you prefer

Comment (carmocca) [reply pytorch_lightning/loops/optimization/manual_loop.py:59]: i'd remove it - generally not a fan of unused and undocumented code

**Review by carmocca — APPROVED** [review]

**Review by ananthsub — APPROVED** [review]

**Review by s-rog — APPROVED** [review]

**Review by SkafteNicki — APPROVED** [review]
